zio-http/src/test/scala/zhttp/service/ServerConfigSpec.scala

```diff
@@ -0,0 +1,56 @@
+package zhttp.service
+
+import io.netty.handler.codec.http.{HttpHeaderNames, HttpHeaderValues, HttpVersion}
+import zhttp.http.{!!, Header, Http, Method}
+import zhttp.internal.{AppCollection, HttpRunnableSpec}
+import zhttp.service.server._
+import zio.test.Assertion.{equalTo, isNone, isSome}
+import zio.test.assertM
+
+object ServerConfigSpec extends HttpRunnableSpec(8088) {
+
+  /*
+    From github discussion (Keep Alive Test case 2, Scenario 1)
+
+      https://github.com/dream11/zio-http/discussions/592
+   */
+  def keepAliveSpec = suite("KeepAlive Test cases") {
+    suite("Connection: close request header test with Server KeepAlive ENABLED") {
+      val connectionCloseHeader = Header(HttpHeaderNames.CONNECTION.toString, HttpHeaderValues.CLOSE.toString)
+      val app1                  = Http.empty
+      testM(
+        "Http 1.1 WITHOUT 'Connection: close' in the request header SHOULD respond WITHOUT 'Connection: close' in the response header, indicating re-use",
```

Comment:
Tests should have a concise description.

Reply:
Made description much shorter.